src/api.rs

```diff
@@ -194,6 +194,12 @@ pub struct AuthenticatedApi<'a> {
     api: &'a Api,
 }
 
+pub struct RegionSpecificApi<'a, 'b> {
```

Comment:
its a bit weird that this has two lifetimes. I believe you can just get along with a single one.

Reply:
Okay, I will try it out!